homeassistant/components/fyta/quality_scale.yaml

```diff
@@ -0,0 +1,90 @@
+rules:
+  # Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
+  runtime-data: done
+  test-before-setup: done
+  appropriate-polling: done
+  entity-unique-id: done
+  has-entity-name: done
+  entity-event-setup: done
+  dependency-transparency: done
+  action-setup:
+    status: exempt
+    comment: No custom action.
+  common-modules: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  docs-actions:
+    status: exempt
+    comment: No custom action.
+  brands: done
+
+  # Silver
+  config-entry-unloading: done
+  log-when-unavailable: done
+  entity-unavailable: done
+  action-exceptions:
+    status: exempt
+    comment: No custom action.
+  reauthentication-flow: done
+  parallel-updates:
+    status: exempt
+    comment: |
+      Coordinator and only sensor platform.
+
+  test-coverage: done
+  integration-owner: done
+  docs-installation-parameters: done
+  docs-configuration-parameters:
+    status: exempt
+    comment: |
+      No options flow.
+
+  # Gold
+  entity-translations: done
+  entity-device-class: done
+  devices: done
+  entity-category: done
+  entity-disabled-by-default:
+    status: exempt
+    comment: No noisy entities.
+  discovery:
+    status: exempt
+    comment: Fyta device cannot be discovered.
+  stale-devices: done
+  diagnostics: done
+  exception-translations: done
+  icon-translations: done
+  reconfiguration-flow:
+    status: exempt
+    comment: No configuration besides credentials.
+  dynamic-devices: done
+  discovery-update-info:
+    status: exempt
+    comment: Fyta cannot be discovered.
```

Comment:
Fyta can be discovered but has no need for a local connection thus does not have to update info